stream recap: 5 Jan (#159)
jjon-saxton/yaya-world: content/en/news/2026/stream-recap-5-jan.org

#+begin_export html
<iframe width="560" height="315" src="https://www.youtube.com/embed/7NGpi6j7714?si=MtyvoKdFviPR-fy8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
#+end_export
** Dark Cloud
I leveled a couple weapons and then I tried again and then I failed, but I ran into a new strategy. Just kinda tried something and it seemed to have worked. Now the Dark Genie never actually let us kick him, but I did bonk him good and I think he'll be behaving now! Dark Cloud 2 next time?
#+begin_export html
<iframe width="560" height="315" src="https://www.youtube.com/embed/KaGXPQhGpcE?si=FWH-auTG2a2Cklza" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
#+end_export
** Party Animals (Community Nite)

> We had a very successful community nite. I played a variety of games with my kittens,  B,  and her community. It was a lot of fun and I hope to do more community nites in the future. Hopefully we'll get some more involvement right here on YouTube as well!

Reviewer: just on "YouTube" might be better since this is the blog
```suggestion
We had a very successful community nite. I played a variety of games with my kittens,  B,  and her community. It was a lot of fun and I hope to do more community nites in the future. Hopefully we'll get some more involvement from YouTube as well!
```
